Polygon_mesh_processing/doc/Polygon_mesh_processing/PackageDescription.txt

```diff
@@ -148,6 +148,7 @@ and provides a list of the parameters that are used in this package.
 - `CGAL::Polygon_mesh_processing::corefine()`
 - `CGAL::Polygon_mesh_processing::surface_intersection()`
 - `CGAL::Polygon_mesh_processing::does_bound_a_volume()`
+- `PMPCorefinementNewFaceVisitor`
```

Comment:
This lone concept looks a little strange (other concepts are not listed here), other packages put concepts at the top but it's not the case with PMP (and it probably wouldn't work).